### ISSUE TITLE
accessibility of the buttons at the bottom of the session details doesn't convey the right information.

### ISSUE THREAD

**Issue by Pluu** [DroidKaigi/conference-app-2023#503]
The accessibility of each button should be called as intended.

| Share | Add Calendar | Bookmark |
| :--: | :--: | :--: |
| <img src="https://github.com/DroidKaigi/conference-app-2023/assets/1534926/621a40f1-f066-485d-87a6-77a87b7c2e21" width="300" /> | <img src="https://github.com/DroidKaigi/conference-app-2023/assets/1534926/237539b0-91fa-4a2f-95ba-bce1a1889d6c" width="300" /> | <img src="https://github.com/DroidKaigi/conference-app-2023/assets/1534926/9a73548b-74d6-4968-8eaf-dccbd23e8696" width="300" /> |

**Comment by takahirom** [comment]
We need this! Thanks!

**Comment by Pluu** [comment]
🙋

**Comment by takahirom** [comment]
Assigned! Thanks
